src/posts/keymaps/ulounge.md

```diff
@@ -0,0 +1,24 @@
+---
+author: ulounge
+baseLayouts: [Ergo-L]
+firmwares: [ZMK]
+hasHomeRowMods: true
+hasLetterOnThumb: false
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: true
+isTapDanceEnabled: true
+keybindings: []
+keyboard: Ferris
+keyCount: 34
+keymapImage: https://raw.githubusercontent.com/ulounge/zmk-urchin/refs/heads/master/assets/images/My_Uchin.drawio.svg
+keymapUrl: https://github.com/ulounge/zmk-urchin
+languages: [French, English]
+layerCount: 7
+OS: ["Windows", "Android"]
+stagger: columnar
+summary: "See write up link for more info." 
```

Comment:
```suggestion
summary: "Keymap designed to balance French and English typing efficiency. Thumb-operated sticky, tap-dance  and hold-tap layers provide quick access to accents, shortcuts, callum-style mods, and media controls. See write up link for more info." 
```

Simply saying “see write up for more info” is not very enticing so may I suggest adding a short blurb?

Reply:
Perfect ! Thanks for the input.